frontend/src/components/Dummy.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Dummy = () => <div>Dummy Test</div>;
+
+export default Dummy;
```

Comment:
Ideally if there not more than one export you can remove verbose variable declarations.

It should work

```
export default = () => <div>Dummy Test</div>;
```

Reply:
export default () => <div>Dummy Test</div>; works.